test/RankifyInstance.ts

```
@@ -1,8 +1,8 @@
 import EnvironmentSimulator, { MockVote, ProposalSubmission } from '../scripts/EnvironmentSimulator';
 import { expect } from 'chai';
 import { time } from '@nomicfoundation/hardhat-network-helpers';
-import { DistributableGovernanceERC20, Governor, Rankify, RankifyDiamondInstance } from '../types/';
+import { DistributableGovernanceERC20, Governor, IRankifyInstance, Rankify, RankifyDiamondInstance } from '../types/';
 import { IRankifyInstance, LibCoinVending } from '../types/src/facets/RankifyInstanceMainFacet';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'IRankifyInstance'. Consider to delete it or rename it.

'IRankifyInstance' is defined here:


(lint/suspicious/noRedeclare)